package.json

```diff
@@ -5,44 +5,47 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "rename:demo": "node -e \"require('fs').rename('dist/demo.html', 'dist/index.html', function(err) { if (err) console.log(err); console.log('File demo.html successfully renamed to index.html!') })\"",
-    "build:ci": "vue-cli-service build --target wc --name extension-marketplace && npm run rename:demo",
+    "build:ci": "vue-cli-service build --target wc --name extension-marketplace && cp public/_redirects dist && npm run rename:demo",
```

Comment:
I would use `cpx` command to prevent:

`'cp' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! marketplace@0.1.0 build:ci: `vue-cli-service build --target wc --name extension-marketplace && cp public/_redirects dist && npm run rename:demo`   
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the marketplace@0.1.0 build:ci script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\OndrejCh\AppData\Roaming\npm-cache\_logs\2020-01-21T11_23_58_786Z-debug.log`

On windows OS.